src/rules/czech.toml

```diff
@@ -0,0 +1,16 @@
+allowed_symbols_regex="[A-Za-zěščřžýáíéóďťňĚŠČŘŽÝÁÍÉÓĎŤŇäöüÄÖÜ‚–\\. ]"
```

Comment:
You missed `úůÚŮ`, and I'd perhaps remove `ÄÖÜäöü` - they are only used in foreign names and some might have issues pronouncing them, but don't consider me an authority on this.

Reply:
Thanks, these are all good suggestions, plus the above, I will incorporate them when I have time to make a next run.

Reply:
Added except for Ů that would anyways always get removed as a non-word-leading capital